### PR TITLE
New Necropolis

### PR DESCRIPTION
**Pull request by John-Dang**
## About The Pull Request
This is a complete rework of the Ash Walker Nest, this is the Ash Walker Town, a whole new Necropolis! The temple has been moved to the center of the village, and has been made larger and given two adjoining rooms and an altar, trying to make it feel like a real central temple the tribe is focused around. The clinic has been moved and reworked to be a shaman's hut, with makeshift surgery tools, mourning poultice and aloe. Residential room has been added with small dining table, dirty mattresses and chests with 4 guard/hunter loadouts, 1 architect loadout, and 1 farmer loadout. A commissary has been added, where loot can be stored and organized for those who want to sort the epic gains made from mining and warfare. A field kitchen and workshop are added to the open area of the village to make things feel a bit lived in. The farm has also been reworked, with more plots, seeds for bufo, cotton, bamboo, aloe, towercap and glowcap. A fighting arena has been added with chain link fence stolen from a foreign outpost and a few indicative bone piles. The bodies are replaced with charred skeletons.

image

## Why It's Good For The Game
Reworks the Necropolis to be more of a village, gives more RP opportunity and visual distinction between villagers, gives them more to interact with and more to do.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by remove32** [comment]
just make sure to update the CL section when you open it.
youll probably get some other feedback later as well

**Comment by Helianthus00** [comment]
Overall, I like it. However, I think that giving a crown/throneroom is not exactly fitting in lorewise. This space could be used better. The door on the bottom right room is also blocked by rocks, so that should be changed. Removing the syndicate stuff removes the extra hint over how they got the modern equipment, and I feel like removing that with no analog is a bad idea.

File: tgstation.dme [hunk 1]
```diff
@@ -733,6 +733,7 @@
 #include "code\datums\wounds\pierce.dm"
 #include "code\datums\wounds\slash.dm"
 #include "code\datums\wounds\scars\_scars.dm"
+#include "code\Fulp_Code\datums\lavaland.dm"
```
Comment (SgtHunk) [inline tgstation.dme:736]: This file no longer exists

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Comment by Gurkenglas** [comment]
The malf AI disk gave the station a reason to interact with Ashwalkers through trade or raiding. It lets AI players learn the malf controls before playing as antag. Why remove it?

**Comment by ATH1909** [comment]
"Also, removes the Malf AI disk and syndicate toolbox from the village area."
cringe

as mentioned by gurk, the malf AI disk is fun and lets AIs learn how being a malf AI works without risking an antag round over it, and the syndicate toolbox expands the number of construction projects and surgeries available to the ash walkers immensely.

**Comment by ATH1909** [comment]
also also, will ash walkers still have enough cloth to make some backpacks after this change?

**Comment by John-Dang** [comment]
You raise a good point about the Malf AI disk! I'll re-add it and do another push in a hot minute. Although is replacing the syndicate toolbox with a regular toolbox alright?

**Comment by John-Dang** [comment]
I am going to try and make a new branch since people on the coderbus server told me to because mine is hilariously out of date they say, so I'm going to close this one and try to open on a new, more up to date branch.

**Pull request closed without merging**